### ISSUE TITLE
MWS menu in dashboard not appearing

### ISSUE THREAD

**Issue by natea** [django-oscar/django-oscar-mws#3]
I've added these lines to my settings.py file, but the MWS menu item is not showing up when I go to the dashboard.

```
from oscar import get_core_apps
INSTALLED_APPS = INSTALLED_APPS + ['oscar_mws'] + get_core_apps([...])

from oscar.defaults import *
from oscar_mws.defaults import *

OSCAR_DASHBOARD_NAVIGATION += MWS_DASHBOARD_NAVIGATION
```

Is there something else that I'm missing?

**Comment by roadsideseb** [comment]
That's a good question. As far as I can see, there's nothing wrong with the settings above. I've tried running the sandbox which pretty much includes the same settings and the menu item shows up without problems. Which versions of Django and Oscar are you using?

I remember having some issues previously with an older bug in Django that would load the settings module twice and cause unexpected behavious in `OSCAR_DASHBOARD_NAVIGATION`. Other than that, the only reason I can see for this, is that you your `OSCAR_DASHBOARD_NAVIGATION` is overwritten later in the file or by another import later on. How are you handling your settings?

**Comment by natea** [comment]
I had a similar problem when I was trying to integrate django-oscar-paypal with my Oscar site, and the only way I could figure out how to get it working was to copy the sandbox directory and build from there. So I'll do the same with the MWS sandbox and see if I can get it working that way.

Here's my settings.py file:
https://gist.github.com/natea/40890a503faae448d968

**Comment by roadsideseb** [comment]
I can't see anything obviously wrong with your settings above. The only thing that could have unexpected side effects is the `from integration import *` at the bottom. If you override the `OSCAR_DASHBOARD_NAVIGATION` in the `integration` module, the changes wouldn't take effect. Other than that, I'm not quite sure what the issue could be.

**Comment by natea** [comment]
Yeah, I'm not sure what it is either. I have it sorta working with the Sandbox but haven't tried to get it working again with my existing Django Oscar site. I'll report back how that goes when I cross that bridge.
